### ISSUE TITLE
fortigate.py : get method used on a list 

### ISSUE THREAD

**Issue by Sulray** [vladimirs-git/fortigate-api#19]
Hi ! 

I was trying to use your lib as a connector to my fortigates but I faced an issue.
When I just want to do a get request I end up with the following: 

> File "xxxx/lib/python3.10/site-packages/fortigate_api/fortigate.py", line 301, in get
>     results: LDAny = response_json.get("results") or []
> AttributeError: 'list' object has no attribute 'get'

To get that, I just use in a script your FortigateAPI class on which I call the following:

> response = self.fortigateAPI.rest.get(full_url)

After some debugging I managed to see that response_json was indeed a list so get method is not defined for that.
![image](https://github.com/vladimirs-git/fortigate-api/assets/42110051/e4e90065-e197-417a-b3ad-071c64118dcd)

Would you know what could lead to that ? Is it a bad implementation on my side or a code issue ?

**Comment by vladimirs-git** [comment]
Could you please share the relevant portion of your code where you are making the get request using FortigateAPI library?

The error you've shared indicates that the response_json is expected to be a dictionary, but it seems that in your case, it is a list, leading to the 'AttributeError.'

By examining your code, I can better pinpoint the source of the problem and provide you with a more accurate solution.
Value of full_url could be useful.

**Comment by vladimirs-git** [comment]
Please try new method Fortigate.get_l() in the version 1.3.0. This method returns a list of data exactly as it is in the request.
```python

import logging
from pprint import pprint

from fortigate_api import Fortigate

logging.getLogger().setLevel(logging.DEBUG)

HOST = "host"
USERNAME = "username"
PASSWORD = "password"

fgt = Fortigate(host=HOST, username=USERNAME, password=PASSWORD)


# Get list
output = fgt.get_l(url="/api/v2/monitor/firewall/policy?global=1")
pprint(output)
# [{'build': 2093,
#   'http_method': 'GET',
#   'name': 'policy',
#   'path': 'firewall',
#   'results': [{'active_sessions': 0,
#                'asic_bytes': 0,
#                'asic_packets': 0,
# ...
```

**Comment by Sulray** [comment]
Seems to work fine thanks ! So it means that depending on the endpoint we want to reach we could end up either with a list or not ? Do you know how to determine that ?

**Comment by vladimirs-git** [comment]
Yes, you need to know the data types. I have no idea where to get the schema for all endpoints. Try examples [monitor.py](https://github.com/vladimirs-git/fortigate-api/blob/main/examples/monitor.py). In replies you can find schema. I hope it can help you.
